src/util/is-vod.js

```diff
@@ -0,0 +1,13 @@
+//utility function used to determine if a given service is a vod or a live stream
```

Comment:
Can you rewrite this to be in JSDoc style? Might look something like [this](https://github.com/MemeLabs/Rustla2/blob/2538f8b46f56ba57a25efae8d6fe63c5976407e3/src/util/is-valid-advanced-url.js#L1-L12).

Reply:
Definitely! I've added that to the latest commit, let me know if that works.